go.mod

```diff
@@ -1,14 +1,15 @@
 module github.com/enterprise-contract/ec-policies
 
 go 1.22.7
+toolchain go1.23.4
```

Comment:
Is it okay to bump go version like this?

Reply:
No. We won't be able to build in Konflux :disappointed:

Reply:
Here it might be okay because this repository is not used as a go dependency

Reply:
Ah that's true. Let's merge it.